osu.Game/Screens/Edit/Submission/SubmissionStageProgress.cs

```diff
@@ -33,8 +39,19 @@ public partial class SubmissionStageProgress : CompositeDrawable
         [Resolved]
         private OsuColour colours { get; set; } = null!;
 
+        private Sample progressSample = null!;
+
+        private const int stage_done_sample_count = 4;
+        private Sample stageDoneSample = null!;
+
+        private Sample errorSample = null!;
+        private Sample cancelSample = null!;
```

Comment:
All of these samples should be declared nullable (`Sample?`) as they are in most other places